app/views/follow_requests/destroy.js.erb

```diff
@@ -0,0 +1,4 @@
+var unfollow_btn = $(".<%=dom_id(@follow_request.recipient)%>_<%=dom_id(@follow_request.sender)%>_form")
```

Comment:
you are forgetting a semi-colon here so your follow/unfollow button is not working